package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "braintree-angular",
-  "version": "1.2.1",
+  "version": "1.2.2",
   "description": "Use Braintree in your Angular app",
-  "main": "index.js",
+  "main": "dist/braintree-angular.js",
```

Comment:
I am not familiar enough with with either bower or npm to know exactly what these are for, but I know that wiredep looks for the main property to know what to dependency to use.
As for npm, I just read about its main property [here](https://docs.npmjs.com/files/package.json#main) and I am inclined to say that my change is correct. Feel free to disagree. I did not make that change for functionality, just consistency.